frontend/workflows-lib/tests/components/WorkflowAccordion.test.tsx

```diff
@@ -1,13 +1,15 @@
 import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
```

Comment:
and here

Reply:
Yeah, weirdly `jest-dom` isn't actually `jest` specific and is recommended as the to test the dom by `testling-library`, see: https://testing-library.com/docs/svelte-testing-library/setup/#vitest